frappe/model/base_document.py

```diff
@@ -399,6 +399,9 @@ def get_valid_dict(
 							eval_locals={"doc": self},
 						)
 
+				if frappe.db.db_type == "postgres":
```

Comment:
Could you add a comment with a short summary of why its required, so people can understand just by looking at the code rather than searching for the commit?